src/debugpy/common/log.py

```diff
@@ -332,7 +332,7 @@ def report_paths(get_paths, label=None):
     site_packages = [
         p
         for p in sys.path
-        if os.path.exists(p) and os.path.basename(p) == "site-packages"
+        if os.path.exists(p) and os.path.basename(p) == compat.filename_str("site-packages")
```

Comment:
`sys.path` entries can actually be unicode (as was the case for the recent issue on Maya), so, you may want to do `filename_str` on `basename` too -- this use-case probably warrants a test too.

Reply:
In this case, it wouldn't be a problem. The reason why it was a problem before is because the literal was `unicode` on Python 2, and that caused the left side to be implicitly converted to Unicode also, with "ascii" as codec - which breaks if it actually has any non-ASCII symbols in it. Now that the right side is always `bytes` on Python 2, if the left side is `unicode`, it's the right side that will be auto-decoded as "ascii" - which is fine, because it really is ASCII.

Reply:
Thanks for the explanation (it didn't occur to me that having ascii bytes on the right side would never raise an exception).